homeassistant/components/deconz/binary_sensor.py

```diff
@@ -76,6 +88,30 @@
 }
 
 
+BINARY_SENSOR_DESCRIPTIONS = [
+    DeconzBinarySensorDescription(
+        device_property="tampered",
+        suffix="Tampered",
+        update_key="tampered",
+        entity_description=BinarySensorEntityDescription(
+            key="tamper",
+            device_class=BinarySensorDeviceClass.TAMPER,
+            entity_category=EntityCategory.DIAGNOSTIC,
+        ),
+    ),
+    DeconzBinarySensorDescription(
+        device_property="low_battery",
+        suffix="Low Battery",
+        update_key="lowbattery",
+        entity_description=BinarySensorEntityDescription(
+            key="low_battery",
+            device_class=BinarySensorDeviceClass.BATTERY,
+            entity_category=EntityCategory.DIAGNOSTIC,
+        ),
+    ),
+]
```

Comment:
Usually (`git grep RequiredKeysMixin`) these are done as more flat structure that keeps all the key/values at a single level.

Reply:
Thanks @bdraco , I'll change it.